p4-16/spec/P4-16-spec.mdk

```diff
@@ -1797,7 +1797,7 @@ restricted places in P4 programs.
 
 ### The error type
 
-The error type contains opaque values that can be used to signal
+The error type contains opaque distinct values that can be used to signal
```

Comment:
This might be an improvement to the spec, but it seems unrelated to casting?

Reply:
Yes, it's not related to casting. I just fixed little things that came up as I was working on the formalization.